File: frontend/src/types/facebook/facebook-user-page.ts
export interface Category {
    id: string;
    name: string;
}

export interface FacebookPagePicture {
    height: number;
    isSilhouette: boolean;
    url: string;
    width: number;
}

export interface FacebookUserPage {
    category: string;
    categoryList: Category[];
    name: string;
    pageId: string;
    tasks: string[];
    pagePicture?: FacebookPagePicture;
}

File: backend/KanriSocial/src/SocialMediaManager.Shared/Dtos/Facebook/FacebookUserPage.cs
namespace SocialMediaManager.Shared.Dtos.Facebook;

public record FacebookUserPage
{
    public string Category { get; init; }
    public List<Category> CategoryList { get; init; }
    public string Name { get; init; }
    public string PageId { get; init; }
    public List<string> Tasks { get; init; }
    public FacebookPagePicture? PagePicture { get; init; } = null;
};